src/components/MessagesList/MessagesList.vue

```diff
@@ -615,36 +610,67 @@ export default {
 			return null
 		},
 
+		/**
+		 * Temporarily highlight the given message id with a fade out effect.
+		 *
+		 * @param {number} messageId message id
+		 * @param {boolean} smooth true to smooth scroll, false to jump directly
+		 * @param {boolean} highlightAnimation true to highlight and set focus to the message
+		 * @return {boolean} true if element was found, false otherwise
+		 */
+		focusMessage(messageId, smooth = true, highlightAnimation = true) {
+			const element = document.getElementById(`message_${messageId}`)
+			if (!element) {
+				// Message id doesn't exist
+				console.warn('Message to focus not found in DOM', messageId)
+				return false // element not found
+			}
+
+			console.debug('Scrolling to a focused message programmatically')
+			this.isFocusingMessage = true
+
+			this.$nextTick(async () => {
```

Comment:
```suggestion
```

As you checking the element in the DOM syncronously, you shouldn't wait for the next tick